public/index.html

```diff
@@ -93,148 +93,169 @@
+                                <img id="nebula-1" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-2" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-3" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
                     </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-1" class="img-responsive" src="" alt="">
-                        </a>
+                    <div class="row-centered">
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-4" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-5" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-6" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
                     </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-10" class="img-responsive" src="" alt="">
-                        </a>
+                    <div class="row-centered">
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-7" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-8" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-9" class="img-responsive" src="" alt="">
```

Comment:
Whooaaaa... Isn't there a way to generate this programmatically? (The nebula-x's, that is.)